apps/karma.conf.js

```diff
@@ -65,6 +71,7 @@ module.exports = function (config) {
 
     // Configures the karma server to map urls to local file paths.
     proxies: {
+      '/assets/images/': '/base/build/images/',
```

Comment:
Can you clarify exactly what functionality this is providing in our build process, and why we don't need to do this for the other subdirectories of the apps package? I'm still unclear on the precise details of how these images are handed off from apps to dashboard.